README.md

```diff
@@ -1 +1,25 @@
-# knotx-adapter-archetype
\ No newline at end of file
+![Knot.x logo](https://avatars2.githubusercontent.com/u/24564424?v=3&s=400)
+
+# Knot.x Extension Maven Archetype
+
+Use this archetype to generate a custon _Knot.x_ extension.
```

Comment:
@Skejven fair enough. Added Action Adapter to the TODO list in the opening comment of this issue.